app/src/renderer/components/common/TmSessionHardware.vue

```diff
@@ -8,7 +8,9 @@
         <div class="tm-session-title">
           Sign In
         </div>
-        <a @click="help"> <i class="material-icons">help_outline</i> </a>
+        <a @click="$store.commit(`toggleSessionModal`, false)">
```

Comment:
is this testable?

Reply:
hypothetically

Reply:
is it tested?

Reply:
i just added a test

Reply:
I think creating a method would have been the cleaner solution as it is done in most of the other components.

Reply:
yes, you are right. please fix in #2155 🙏 🐱